libs/components/datetime/src/lib/modules/date-range-picker/date-range.service.spec.ts

```diff
@@ -43,7 +43,8 @@ describe('Date range service', function () {
   });
 
   it('should handle calculator not found', function () {
-    service.getCalculatorById(5000).catch((error) => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
For this particular issue, I'd rather not suppress the warning from ESLint since we want to figure out a solution for the calculator ID type.

Reply:
Done